s4e-backend/src/main/java/pl/cyfronet/s4e/license/types/PrivateLicense.java

```diff
@@ -6,19 +6,34 @@
 
 import static pl.cyfronet.s4e.security.AppUserDetailsUtil.isAdmin;
 import static pl.cyfronet.s4e.security.SecurityConstants.LICENSE_READ_AUTHORITY_PREFIX;
+import static pl.cyfronet.s4e.security.SecurityConstants.LICENSE_WRITE_AUTHORITY_PREFIX;
+
+public class PrivateLicense implements WritableLicense {
+    @Override
+    public boolean canWrite(Product product, AppUserDetails userDetails) {
+        if (isAdmin(userDetails)) {
```

Comment:
Wouldn't be better to `return isAdmin(...) ? true : hasWriteLicense(userDetails, product);`?